process/transaction/shardProcess.go

```diff
@@ -298,7 +304,14 @@ func (txProc *txProcessor) executingFailedTransaction(
 		return nil
 	}
 
-	txFee := txProc.economicsFee.ComputeTxFee(tx)
+	txFee := txProc.economicsFee.ComputeFeeForProcessing(tx, tx.GasLimit)
+	if txProc.enableEpochsHandler.IsFlagEnabled(common.FixRelayedMoveBalanceFlag) {
```

Comment:
move the common code and use a common function. This is the exactly same lines of codes as in baseProcess.